finality-aleph/src/party.rs

```diff
@@ -216,36 +351,39 @@ where
     SC: SelectChain<B> + 'static,
     NumberFor<B>: From<u32>,
 {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Ouch, same comment as before.

Reply:
Here the solution is to remove `authority` param as it may be read from `auth_keystore`.

Reply:
Yeah, I think @mike1729 is correct, but then I would have to place this low level line `let auth = SyncCryptoStore::ed25519_public_keys(&*keystore, finality_aleph::KEY_TYPE)[0];` somewhere here -- which well I can do, but if it's only playing around this arbitrary limit of 7 parameters then I have my doubts :P.

Reply:
No, just `let authority = auth_keystore.authority()` :)

Reply:
Ok, sorry for that, you are right! Fixed and removed clippy silencer.